cases/spectypes.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  parseSafe,
+  parseStrict,
+  assertLoose,
+} from '../compiled/spectypes/build';
```

Comment:
What is the reason for the `compiled` directory?

IMHO it would look cleaner to readers if all the code did reside in `cases/spectypes/` - is that possible?

Reply:
> IMHO it would look cleaner to readers if all the code did reside in cases/spectypes/
Agreed

Reply:
Ok, I have changed folder structure for spectypes.